.gitignore

```diff
@@ -2,3 +2,4 @@ target
 Cargo.lock
 
 .cargo/config.toml
+.idea
```

Comment:
Best to leave IDE specific files out - individuals can add these lines to their global gitignore file.